templates/bluemix/manifest.yml

```diff
@@ -0,0 +1,16 @@
+declared-services:
```

Comment:
I will keep only the bare basics for now, and remove it. Let's develop the interface and content of the file progressively.